onnx/backend/test/case/node/reduce_log_sum.py

```diff
@@ -0,0 +1,39 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+
+import numpy as np  # type: ignore
+
+import onnx
+from ..base import Base
+from . import expect
+
+class ReduceLogSum(Base):
+
+    @staticmethod
+    def export():
+        node = onnx.helper.make_node(
+                'ReduceLogSum',
+                inputs=['data'],
+                outputs=["reduced"],
+                axes=[2, 1],
+                keepdims=0
+            )
+        data = np.random.ranf([3, 4, 5]).astype("float32")
+        reduced = np.log(np.sum(data, axis=(2, 1), keepdims=False))
```

Comment:
right now, both cases are descending order, we can have one case with ascending axes.